keylime/ima/ima.py

```diff
@@ -245,8 +239,8 @@ def _process_measurement_list(
     if lists is not None:
```

Comment:
I think we can rename `lists` to `ima_policy` here. 

Also can we eliminate the `json.loads(lists)`? This function should be only called with an already parsed policy and if not we should fix that.

Reply:
@THS-on Agreed on both counts. I've included a stub function called `deserialize_ima_policy` that is intended to translate the IMA policy DB object into something usable by the verifier; it doesn't do anything right now (the intention was to use it to load DSSE-formatted IMA policies, but that didn't make it in), but I could adapt it to return JSON-formatted policies. That would then be passed down the chain and we could eventually drop that `json.loads` call.

Reply:
@THS-on From digging into the code, it seems like there are some tests that pass JSON-formatted strings to `process_measurement_list()`, but that's the only scenario I'm aware of where that function isn't processing a JSON dictionary. I think we can remove those tests.

Reply:
Yes and make sure that when the data is loaded from the database is a dictionary and not a string. This normally happens on startup of the verifier.

Reply:
@THS-on Yes - you can see that any time we ask for a policy from the database, it has to go through `ima.deserialize_ima_policy()`, which at current just calls `json.loads()` on the database object - so it will always be JSON after. In the future we can add to this function to support things like DSSE in the DB.